### DISCUSSION TITLE
Not work with nestjs / ng-universal

### DISCUSSION THREAD

**Discussion by max5432112345** [mikro-orm/mikro-orm#3456] [Q&A]
**Describe the bug**
Not work with nestjs and angular universal https://github.com/nestjs/ng-universal

./node_modules/@mikro-orm/core/utils/Utils.js:951:13-50 - Warning: Module not found: Error: Can't resolve '../../package.json' in '/home/xxx/Projects/angular/selbi/node_modules/@mikro-orm/core/utils'

and other such message for pg, mysql and other
**Stack trace**
```
...
```

**To Reproduce**
1. ...
2. ...
3. ...

**Expected behavior**

**Additional context**

**Versions**

| Dependency | Version |
| - | - |
| node | 16 |
| typescript | 4 |
| mikro-orm | 5 |
| your-driver | mysql |

**Comment by max5432112345** [comment]
Is need to instal angular and run ng add @nestjs/ng-universal. And add mikro orm to nestjs. And shit will produce :1st_place_medal: )

**Reply by khmm12** [reply]
@max5432112345 IMO it makes sense to provide a minimal working repository where the issue is clearly reproducible.

**Reply by max5432112345** [reply]
my life will pass if I produce reproductions for whatever shit

**Reply by max5432112345** [reply]
shit on github are very much
